harvard-lts/etd_alma_drs_holding_service
# Etd 337: ETDAlmaDRSHoldingService - using the Alma API

**ETDAlmaDRSHoldingService - using the Alma API**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-337
https://at-harvard.atlassian.net/browse/ETD-348  (tracing)

# What does this Pull Request do?
uses the alma api to update holdings in alma that are not in dash.

# How should this be tested?

0) get all alma creds
1) start the container
2) enter the container then run`pytest`. all 10 tests should pass.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
